frontend/src/components/main-page/groups-tree-page/filter/models.ts

```diff
@@ -58,12 +68,26 @@ export const QUERY_KEY_TO_SERVICEDOC_MAP: QueryKeyToServiceDocKeyMap = {
 
   responsibleteam: 'responsibleTeam',
   responsible: 'responsibles',
-};
+} as const;
```

Comment:
I wanted to use the `satisfies` operator that was introduced in TS 4.9. However, it seems like the babel version installed with `create-react-app` does not support that yet.